docs/REST/authentication.md

```diff
@@ -530,4 +530,43 @@ $secret = "<Secret>"
 
 $headers = Sign-Request $uri.Authority $method $uri.PathAndQuery $body $credential $secret
 Invoke-RestMethod -Uri $uri -Method $method -Headers $headers -Body $body
+```
+### Bash
```

Comment:
>Bash [](start = 4, length = 4)

You might add *Prerequisites* for non-built in commands so people will know what they need to try it.